rotkehlchen/db/dbhandler.py

```diff
@@ -229,8 +231,13 @@ def db_tuple_to_str(
     raise AssertionError('db_tuple_to_str() called with invalid tuple_type {tuple_type}')
 
 
+def _load_timestamp(str_timestamp: str) -> Timestamp:
+    return Timestamp(int(datetime.datetime.strptime(str_timestamp, '%Y-%m-%d %H:%M:%S').timestamp()))  # noqa: E501
+
```

Comment:
We already have this at https://github.com/rotki/rotki/blob/develop/rotkehlchen/utils/misc.py#L49

Reply:
Yeah, thanks!